mqtt/mqtt-bridge/tests/config.json

```diff
@@ -4,11 +4,15 @@
     "moduleid": "mymodule",
```

Comment:
From Denis: All these fields are only relevant to upstream, so maybe we should move to the upstream section.

Reply:
they are actually retrieved from env variables so they are at top level, here are just for configuring tests